packages/integrations/image/README.md

```diff
@@ -0,0 +1,171 @@
+  
+  ```sh
+  # Using NPM
+  npx astro add image
+  # Using Yarn
+  yarn astro add image
+  # Using PNPM
+  pnpx astro add image
+  ```
+  
+Then, restart the dev server by typing `CTRL-C` and then `npm run astro dev` in the terminal window that was running Astro.
+  
+Because this command is new, it might not properly set things up. If that happens, [feel free to log an issue on our GitHub](https://github.com/withastro/astro/issues) and try the manual installation steps below.
+</details>
+
+<details>
+  <summary>Manual Install</summary>
+  
+<br/>
+  
+First, install the `@astrojs/image` package using your package manager. If you're using npm or aren't sure, run this in the terminal:
+```sh
+npm install @astrojs/image
+```
+Then, apply this integration to your `astro.config.*` file using the `integrations` property:
+
+__astro.config.mjs__
+
+```js
+import image from '@astrojs/image';
+
+export default {
+  // ...
+  integrations: [image()],
+}
+```
+  
+Then, restart the dev server.
+</details>
+
+## Usage
+
+The built-in `<Image />` component is used to create an optimized `<img />` for both remote images hosted on other domains as well as local images imported from your project's `src` directory.
+
+The included `sharp` transformer supports resizing images and encoding them to different image formats. Third-party image services will be able to add support for custom transformations as well (ex: `blur`, `filter`, `rotate`, etc).
+
+## Configuration
+
+The intergration can be configured to run with a different image service, either a hosted image service or a full image transformer that runs locally in your build or SSR deployment.
+
+There are currently no other configuration options for the `@astrojs/image` integration. Please [open an issue](https://github.com/withastro/astro/issues/new/choose) if you have a compelling use case to share.
+
+<details>
+  <summary><strong>config.serviceEntryPoint</strong></summary>
+  
+  <br/>
+  
+  The `serviceEntryPoint` should resolve to the image service installed from NPM. The default entry point is `@astrojs/image/sharp`, which resolves to the entry point exported from this integration's `package.json`.
+
+```js
+// astro.config.mjs
+import image from '@astrojs/image';
+
+export default {
+  integrations: [image({
+    // Example: The entrypoint for a third-party image service installed from NPM
+    serviceEntryPoint: 'my-image-service/astro.js'
+  })],
+}
+```
+</details>
+
+## Examples
+
+<details>
+  <summary><strong>Local images</strong></summary>
+  
+  <br/>
+  
+  Image files in your project's `src` directory can be imported in frontmatter and passed directly to the `<Image />` component. All other properties are optional and will default to the original image file's properties if not provided.
+
+```html
+---
+import Image from '@astrojs/image/Image.astro';
```

Comment:
Having `@astrojs/image/components` use named exports would be more future-proof.

I'd prefer the following:

```suggestion
import { Image } from '@astrojs/image/components';
```

Reply:
Yep that's a great call, old habits die hard 😄